src/test/java/com/stripe/model/IssuerFraudRecordTest.java

```diff
@@ -0,0 +1,70 @@
+package com.stripe.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.exception.StripeException;
+import com.stripe.net.APIResource;
+import com.stripe.net.LiveStripeResponseGetter;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class IssuerFraudRecordTest extends BaseStripeTest {
+  @Before
+  public void mockStripeResponseGetter() {
+    APIResource.setStripeResponseGetter(networkMock);
+  }
+
+  @After
+  public void unmockStripeResponseGetter() {
+    /* This needs to be done because tests aren't isolated in Java */
+    APIResource.setStripeResponseGetter(new LiveStripeResponseGetter());
+  }
+
+  @Test
+  public void testDeserialize() throws IOException {
+    String json = resource("issuer_fraud_record.json");
+    IssuerFraudRecord ifr = APIResource.GSON.fromJson(json, IssuerFraudRecord.class);
+    assertEquals("issfr_123", ifr.getId());
+    assertEquals("issuer_fraud_record", ifr.getObject());
+    assertEquals(1525167901, ifr.getCreated().longValue());
+    assertEquals(1501718400, ifr.getPostDate().longValue());
+    assertEquals("ch_123", ifr.getCharge());
+    assertEquals("misc", ifr.getFraudType());
+    assertTrue(ifr.getLivemode());
+  }
+
+  @Test
+  public void testDeserializeExpandCharge() throws IOException {
```

Comment:
Would you mind changing this to `testDeserializeWithExpansions` and renaming the file from `issuer_fraud_record_expansion.json` to `issuer_fraud_record_expansions.json`? (Just adding an "s" for consistency with others in there.)

I realize there appears to be zero rhyme or reason behind the myriads of conflicting ideas in here, but I kind of want to trend us to a place where most resources just have two deserialization tests: one that checks their basic non-expanded form, and one that checks them with every single available expansion. This will generally mean less code to update when new expansions are added.